tests/meta/orders/test_meta_orders_api.py

```diff
@@ -85,33 +114,25 @@ def test_submit(
                 assert link_obj.sample.original_ticket == ticket_id
 
 
-def monkeypatch_process_lims(monkeypatch, order_data):
-    lims_project_data = {"id": "ADM1234", "date": dt.datetime.now()}
-    lims_map = {sample.name: f"ELH123A{index}" for index, sample in enumerate(order_data.samples)}
-    for submitter in SUBMITTERS:
-        monkeypatch.setattr(
-            f"cg.meta.orders.{submitter}.process_lims",
-            lambda **kwargs: (lims_project_data, lims_map),
-        )
-
-
 @pytest.mark.parametrize(
     "order_type",
     [OrderType.MIP_DNA, OrderType.MIP_RNA, OrderType.BALSAMIC],
 )
+@patch("cg.clients.freshdesk.freshdesk_client.FreshdeskClient.create_ticket")
```

Comment:
Is there a reason why you moved the mocking to a decorator instead of a `with` statement? I think it is clearer using `with`, but is a personal opinion